tensorflow/core/util/cuda_kernel_helper.h

```diff
@@ -23,10 +23,82 @@ limitations under the License.
 #include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor"
 #include "tensorflow/core/platform/types.h"
 
+// Usage of GetCudaLaunchConfig, GetCuda2DLaunchConfig, and GetCuda3DLaunchConfig:
+//
+// There are two versions of GetCudaLaunchConfig and GetCuda2DLaunchConfig, one
```

Comment:
It would be less confusing if the two versions had different names, instead of overloading the same symbol.